bin/ovpn_getclient

```diff
@@ -51,3 +75,29 @@ fi
 if [ -n "$OVPN_MTU" ]; then
     echo "tun-mtu $OVPN_MTU"
 fi
+}
+
+dir="$OPENVPN/clients/$cn"
+case "$parm" in
+    "separated")
+        mkdir -p "$dir"
+        get_client_config "$parm" > "$dir/${cn}.ovpn"
+        cp "$EASYRSA_PKI/private/${cn}.key" "$dir/${cn}.key"
+        cp "$EASYRSA_PKI/ca.crt" "$dir/ca.crt"
+        cp "$EASYRSA_PKI/issued/${cn}.crt" "$dir/${cn}.crt"
+        cp "$EASYRSA_PKI/dh.pem" "$dir/dh.pem"
+        cp "$EASYRSA_PKI/ta.key" "$dir/ta.key"
+        ;;
+    "combined")
+        get_client_config "combined"
+        ;;
+    "combined-save")
+        get_client_config "combined" > "$dir/${cn}-combined.ovpn"
+        ;;
+    *)
+        >&2 echo "This script can produce the client configuration in to formats."
+        >&2 echo "    1. combined: All needed configuration and cryptographic material is in one file (Use \"combined-save\" to write the configuration file in the same path as the separated parameter does)."
+        >&2 echo "    2. separated: Separated files."
+        >&2 echo "Please specific one of those options as second parameter."
+        ;;
```

Comment:
I agree with redirecting stderr, but putting the redirection at the beginning of the line feels strange.  Most people expect IO redirection at the end of lines by convention.  Approach of least astonishment is preferred here.